python/SocketMessages.py

```diff
@@ -391,137 +389,137 @@ def dump(self):
-        raise ValueError(f"Runtime status is not of type int: {runtime_status}")
-    if runtime_status not in lookup_runtime_state_types:
-        raise ValueError(f"Runtime status is not a known state: {runtime_status}")
-    return lookup_runtime_state_types[runtime_status]
-
-
-def ensure_type_of_robot_mode(robot_mode: any) -> RobotModeTypes:
-    if not isinstance(robot_mode, int):
-        raise ValueError(f"Robot mode is not of type int: {robot_mode}")
-    if robot_mode not in lookup_robot_mode_types:
-        raise ValueError(f"Robot mode is not a known state: {robot_mode}")
-    return lookup_robot_mode_types[robot_mode]
-
-
-def ensure_type_of_joints(joints: any) -> JointState:
-    if not isinstance(joints, list):
-        raise ValueError(f"Joints are not of type list: {joints}")
-    if len(joints) != 6:
-        raise ValueError(f"Joints are not of length 6: {joints}")
-    for joint in joints:
-        if not isinstance(joint, float):
-            raise ValueError(f"Joint is not of type float: {joint}")
-    return JointState(joints)
-
-
-def ensure_type_of_tcp_pose(tcp_pose: any) -> TCPPoseState:
-    if not isinstance(tcp_pose, list):
-        raise ValueError(f"TCP is not of type list: {tcp_pose}")
-    if len(tcp_pose) != 6:
-        raise ValueError(f"TCP is not of length 6: {tcp_pose}")
-    for tcp_value in tcp_pose:
-        if not isinstance(tcp_value, float):
-            raise ValueError(f"TCP value is not of type float: {tcp_value}")
-    return TCPPoseState(tcp_pose)
-
-
-def ensure_type_of_tcp_speed(tcp_speed: any) -> TCPSpeedState:
-    if not isinstance(tcp_speed, list):
-        raise ValueError(f"TCP speed is not of type list: {tcp_speed}")
-    if len(tcp_speed) != 6:
-        raise ValueError(f"TCP speed is not of length 6: {tcp_speed}")
-    for tcp_value in tcp_speed:
-        if not isinstance(tcp_value, float):
-            raise ValueError(f"TCP speed value is not of type float: {tcp_value}")
-    return TCPSpeedState(tcp_speed)
-
-
-def ensure_type_of_tcp_force(tcp_force: any) -> TCPForceState:
-    if not isinstance(tcp_force, list):
-        raise ValueError(f"TCP force is not of type list: {tcp_force}")
-    if len(tcp_force) != 6:
-        raise ValueError(f"TCP force is not of length 6: {tcp_force}")
-    for tcp_value in tcp_force:
-        if not isinstance(tcp_value, float):
-            raise ValueError(f"TCP force value is not of type float: {tcp_value}")
-    return TCPForceState(tcp_force)
-
-
-def ensure_type_of_tcp(tcp_pose: any, tcp_speed: any, tcp_force: any) -> TCPState:
-    return TCPState(ensure_type_of_tcp_pose(tcp_pose), ensure_type_of_tcp_speed(tcp_speed),
-                    ensure_type_of_tcp_force(tcp_force))
-
-
-def ensure_type_of_payload(payload: any) -> float:
-    if not isinstance(payload, float):
-        raise ValueError(f"Payload is not of type float: {payload}")
-    return payload
+# class TransmittedInformationOptions(Enum):
```

Comment:
I dont expect to use it.
We are receiving all the data as strings now. We have planned to do the parsing of those strings on the frontend.

I think we should just delete these lines